src/index.html

```diff
@@ -15,8 +15,102 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="index.html"
+        class="logo-reference"
+      >
+        <img
+          src="./images/logo.png"
+          alt="logo"
+          class="logo-img"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav-ul">
+          <li class="nav-li nav-li-1">
+            <a
+              href="index.html"
+              class="nav-li nav-li-1 nav-li-reference nav-li-reference-1 is-active"
+            >
+              Apple
+            </a>
```

Comment:
```suggestion
          <li class="nav__item">
            <a
              href="index.html"
              class="nav__link is-active"
            >
              Apple
            </a>
```
<img width="1143" alt="image" src="https://github.com/mate-academy/layout_moyo-header/assets/70220667/ff00e911-4a8b-45fb-b824-008505e692b2">